packages/maps/src/components/MapView.tsx

```diff
@@ -77,6 +78,13 @@ const MapViewF = <T extends object>({
   const delayedRegionValue = useDebounce(currentRegion, 300);
   const contextDelayedRegionValue = useDebounce(currentRegion, 50);
 
+  if (mapType === "mutedStandard" && Platform.OS === "android") {
+    console.warn(
+      "Map type 'mutedStandard' is not supported on Android. Defaulting to 'standard'"
+    );
+    mapType = "standard";
```

Comment:
I think we should provide some default mapType instead of mutate prop directly like this

Reply:
Sure, I can do that